### DISCUSSION TITLE
Cannot download/convert videos

### DISCUSSION THREAD

**Discussion by Andrewczita** [aclap-dev/video-downloadhelper#1400] [Help and Q&A]
### Your question

Hi, I can't download the videos on https://academy.chinafix.tech/. I've tried every way and reported it several times. If you can solve this problem I will appreciate it. I have the premium version.

### Addon & CoApp version and details

Version 9.1.1.0 (stable), target google, locale it.
App integrativa installata (v2.0.19) 
![Uploading Screenshot 2024-10-25 144808.png…]()

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

The convert functions of previous versions of VDH have not been migrated forward to the current VDH 9 versions.  There's a discussion in here in which that is discussed, it you look for it hard enough.

As for your download problem, I can't help you.  Like I said, I don't work for VDH.

**Comment by Mattmjs** [comment]
Bad luck on that site, every video looks to use encryption. Every segment looks to use a key which you can see in the attached file.
So in that case I don't see any software working for this site, just find and use a screen recorder.
[video_m3u8_drm_6675d506f1849182bd71ef607ac0e6e1.txt](https://github.com/user-attachments/files/17528538/video_m3u8_drm_6675d506f1849182bd71ef607ac0e6e1.txt)

**Reply by Andrewczita** [reply]
thanks guys for your information on using a screen recorder but the course is very long and I needed a specific tool. as Mattmjs stated it's all encrypted.😭
